pysrc/scripts/sampling.py

```diff
@@ -20,34 +23,45 @@ def sample_csv(file_path: str,
     """Sample a single file with a certain method and rate
     and create a new tmp file. Returns the path to the sampled file.
     """
-    data: list[list[str]] = []
-    file_prefix = file_path.rsplit('/', 1)[1].rsplit('.', 1)[0]
-    # Read data
-    with open(file_path, 'r') as file:
-        reader = csv.reader(file, delimiter=';')
-        data = list(reader)
 
-    if config.header:
-        file_header = data[0]
-        data = data[1:]
+    samples: list[list[tuple[str, str, float]]] = []
 
-    num_entries = len(data)
-    num_samples = math.ceil(num_entries * sampling_rate)
+    file_prefix = file_path.rsplit('/', 1)[1].rsplit('.', 1)[0]
+    columns = defaultdict(list)
 
-    new_file_name = f'{file_prefix}_{str(sampling_rate).replace(".", "")}_{sampling_method}.csv'
-    new_file_path = os.path.join(os.getcwd(), config.tmp_folder, new_file_name)
+    with open(file_path, 'r') as f:
+        #TODO Input abhängig machen, was passiert mit Headern
+        reader = csv.reader(f, delimiter=';', escapechar='\\')
+        for row in reader:
+            for i in range(len(row)):
+                columns[i].append(row[i])
 
-    sampling_method_function = sampling_methods_dict[sampling_method]
-    data = sampling_method_function(data, num_samples, num_entries)
+    for col in columns:
 
-    with open(new_file_path, 'w') as file:
-        writer = csv.writer(file, delimiter=';')
         if config.header:
-            writer.writerow(file_header)
+            file_header = columns[col][0]
+            data = columns[col][1:]
+
+        num_entries = len(columns[col])
+        num_samples = math.ceil(num_entries * sampling_rate)
+
+        #rename files column specific
+        new_file_name = f'{file_prefix}_{str(sampling_rate).replace(".", "")}_{sampling_method}_{col+1}.csv'
+        new_file_path = os.path.join(os.getcwd(), config.tmp_folder, new_file_name)
 
-        writer.writerows(data)
+        sampling_method_function = sampling_methods_dict[sampling_method]
+        data = sampling_method_function(columns[col], num_samples, num_entries)
 
-    return new_file_path
+        with open(new_file_path, 'w') as file:
+            writer = csv.writer(file)
+            if config.header:
+                writer.writerow()
```

Comment:
```suggestion
                writer.writerow([])
```

To make clear that you're adding an empty line.